docs/content/users/usage/troubleshooting.md

```diff
@@ -98,6 +98,8 @@ sudo apachectl stop
 
 Here are some of the other common processes that could be using ports 80/443 and methods to stop them.
 
+* macOS Content Filtering: Under "Screen Time" -> "Choose Screen Time content and privacy settings" turn off "Content and Privacy". Then you must reboot. This has been a common issue with macOS Sonoma.
```

Comment:
Might want to un-proper-nounify “content filtering”, use a real arrow, and whittle down slightly for brevity:

```suggestion
* macOS content filtering: Under "Screen Time" → "Choose Screen Time content and privacy settings", turn off "Content and Privacy" and then reboot. This has been a common issue with macOS Sonoma.
```